File: BOJ/10XXX/10797-10부제.js
/**
* ABOUT
*
* DATE: 2021-05-22
* AUTHOR: lovelysi0113
*
* URL: https://www.acmicpc.net/problem/10797
*
* COMMENT:
* 주어진 날짜와 같은 일의 자리의 갯수를 세면 된다
**/

const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let input = [];

rl.on('line', function (line) {
    input.push(line);
})
.on('close', function () {
    let today = parseInt(input[0]);
    let cars = input[1].split(' ').map(el => parseInt(el));
    
    let answer = cars.filter(el => el === today).length;
    console.log(answer);
    
    process.exit();
});

File: BOJ/10XXX/10797-10부제.py
'''/**
* ABOUT
*
* DATE: 2021-05-22
* AUTHOR: lovelysi0113
*
* URL: https://www.acmicpc.net/problem/10797
*
* COMMENT:
* 주어진 날짜와 같은 일의 자리의 갯수를 세면 된다
**/'''

today = int(input())
cars = list(map(int, input().split(' ')))

answer = 0
for car in cars:
    if car == today:
        answer += 1

print(answer)
